Kickstarter-iOS/Views/Controllers/SettingsPrivacyViewController.swift

```diff
@@ -0,0 +1,134 @@
+import KsApi
+import Library
+import Prelude
+import Prelude_UIKit
+import SafariServices
+import Result
+import UIKit
+
+public protocol SettingsPrivacyViewControllerDelegate: class {
+  func notifyStartRequestDataTapped()
```

Comment:
Similar suggestion to the others.

Reply:
I'm not using this one so I'm deleting it. 🤦🏾‍♀️